src/multiparser.js

```diff
@@ -163,6 +165,10 @@ function getText(options, node) {
   return options.originalText.slice(util.locStart(node), util.locEnd(node));
 }
 
+function deIndent(options, doc) {
+  return align(-options.tabWidth, doc);
```

Comment:
The indentation comes from the parent node in the AST, which currently has no knowledge that a child might switch to a different parser.